src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -106,6 +106,15 @@ class MessageHelper {
   static void SendDecryptCertificateToHMI(const std::string& file_name,
                                           ApplicationManager& app_mngr);
 
+  /**
+    * @brief SendGetSystemTimeRequest sends mentioned request to HMI.
+    * @param correlation_id the message correlation id, required for proper
+    * response processing.
+    * @param app_mngr
+    */
+  static void SendGetSystemTimeRequest(const uint32_t correlation_id,
```

Comment:
@AByzhynar please avoid methods with side effects in message handler. Replace it with CreateGetSystemTimeRequest, and let caller to sent it via app_mngr